html/changelogs/lohikar-sparks.yml

```diff
@@ -0,0 +1,7 @@
+author: Lohikar
+delete-after: True
+changes: 
+  - tweak: "Refactored sparks to be much less laggy."
```

Comment:
Changelog is silly. The one tweak note at the top is enough, you can just expand it to say, "Bears and electrical storms should lag less now."